src/Ironclad/WebApi/UsersController.cs

```diff
@@ -362,12 +377,47 @@ public async Task<IActionResult> Delete(string username)
                 NormalizedEmail = username.ToUpper(),
                 NormalizedUserName = username.ToUpper(),
             };
-            
+
             await this.userManager.DeleteAsync(user);
 
             return this.NoContent();
         }
 
+        [HttpDelete("byEmailPattern/{emailPattern}")]
+        public async Task<IActionResult> DeleteUsers(string emailPattern)
```

Comment:
Why do we have to remove clients by email pattern? Do not we know exactly what the user email is?

Reply:
this method was added for clean up after the tests